src/FirePokemon.java

```diff
@@ -0,0 +1,110 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class FirePokemon extends Pokemon{
+
+    private static final String type = "fire";
+
+    private static final List<String> attacks = Arrays.asList("Inferno", "PyroBall", "FireLash", "FlameThrower");
```

Comment:
Je schrijft deze attacks met hoofdletters. In de `PokemonGymImpl` wordt er gecheckt op attacks zonder hoofdletters. Dat levert geen probleem op, wat er gaat nog een `.toLowerCase()` overheen, maar het is wel iets om je van bewust te zijn.